selinux/cockpit.te

```diff
@@ -49,8 +49,15 @@ can_exec(cockpit_ws_t,cockpit_ws_exec_t)
 # systemd can execute cockpit-session
 can_exec(init_t,cockpit_session_exec_t)
 
-# cockpit-ws can execute cockpit-session
+# cockpit-ws can execute cockpit-{ssh,session}
 can_exec(cockpit_ws_t,cockpit_session_exec_t)
+# ... and the real ssh (through beiboot/ferny)
+gen_require(`
+    type ssh_exec_t;
+')
+can_exec(cockpit_ws_t,ssh_exec_t)
+# cockpit-client-askpass is written as temporary file
+can_exec(cockpit_ws_t,cockpit_tmp_t)
```

Comment:
I think we can take this out again. We now expect the packaged cockpit-beiboot to call the packaged cockpit-askpass, and for the "remote beiboot" case, everything is just in temp files, and cockpit-ws isn't involved.

Reply:
Done.